build.sbt

```diff
@@ -32,28 +34,66 @@ lazy val allSettings = Seq(
         <name>Ólafur Páll Geirsson</name>
         <url>https://geirsson.com</url>
       </developer>
+      <developer>
+        <id>jonas</id>
+        <name>Jonas Fonseca</name>
+        <url>https://github.com/jonas</url>
+      </developer>
     </developers>
 )
 
 lazy val example = project
 
-lazy val metadoc = crossProject
-  .in(file("metadoc"))
+lazy val protobufSettings = Seq(
+  PB.targets.in(Compile) := Seq(
+    scalapb.gen(
+      flatPackage = true // Don't append filename to package
+    ) -> sourceManaged.in(Compile).value./("protobuf")
```

Comment:
That's a good point, we can move the `protobufSettings` and the `metadoc.proto` to a shared `metadoc-core` or similar name. We can leave that out for a separate PR.